### ISSUE TITLE
Error with 2.0

### ISSUE THREAD

**Issue by karensg** [Casecommons/pg_search#341]
I get the following error when I try
`rails g migration <name>`
on Windows 10 with Ruby 2.3. This does not happen with version 1.0.6. Any idea what is wrong?

```
C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require': cannot load such file -- pg_search/migration/associated_against_generator (LoadError)
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `block in require'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:240:in `load_dependency'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/activesupport-4.2.7.1/lib/active_support/dependencies.rb:274:in `require'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/pg_search-2.0.0/lib/pg_search/railtie.rb:10:in `block in <class:Railtie>'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/railtie.rb:228:in `block in run_generators_blocks'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/railtie.rb:245:in `each'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/railtie.rb:245:in `each_registered_block'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/railtie.rb:228:in `run_generators_blocks'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/application.rb:462:in `block in run_generators_blocks'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/engine/railties.rb:13:in `each'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/engine/railties.rb:13:in `each'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/application.rb:462:in `run_generators_blocks'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/engine.rb:461:in `load_generators'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:129:in `generate_or_destroy'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:50:in `generate'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/commands/commands_tasks.rb:39:in `run_command!'
        from C:/Ruby23-x64/lib/ruby/gems/2.3.0/gems/railties-4.2.7.1/lib/rails/commands.rb:17:in `<top (required)>'
        from bin/rails:4:in `require'
        from bin/rails:4:in `<main>'
```

**Comment by illogikal** [comment]
I also get this issue on mac with latest version

**Comment by bhtabor** [comment]
Same here (ruby '2.3.1' and rails '5.0.0.1'). I'm not sure if Rails 5 is supported though. The requirement just says 4.2+.

**Comment by bartboy011** [comment]
Same issue, did some digging and line 10 of pg_search/lib/pg_search/railtie.rb is requiring the missing `pg_search/migration/associated_against_generator`.

**Comment by bartboy011** [comment]
I went ahead an created a pull request for a fixed version:

https://github.com/Casecommons/pg_search/pull/342

**Comment by nertzy** [comment]
Thanks for the pull request. Fixed and released as 2.0.1.

**Comment by bhtabor** [comment]
Thank you @bartboy011
